docs: Add example to `README.md` (#46)
lukso-network/tools-eip191-signer: README.md

const eip191Signer = new EIP191Signer();
```

> ### Example of Signing an LSP6KeyManager relay call

Reviewer: ```suggestion
### Example: signing a LSP6KeyManager relay transaction
```

> The [LSP6-KeyManager](https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#relay-execution) standard uses version 0 for signed messages. Therefore, it should us [`signDataWithIntendedValidator`](#signdatawithintendedvalidator).

Reviewer: ```suggestion
The [LSP6-KeyManager](https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#relay-execution) standard uses version 0 for signed messages. Therefore, it must use [`signDataWithIntendedValidator`](#signdatawithintendedvalidator).
```

> ```js

Reviewer: I would definitely suggest putting an example for ether.js as well, as ether.js is more used now compared to web3js.
